x-pack/platform/plugins/shared/cases/server/routes/api/cases/post_case.ts

```diff
@@ -25,9 +26,10 @@ export const postCaseRoute = createCasesRoute({
     try {
       const caseContext = await context.cases;
       const casesClient = await caseContext.getCasesClient();
-      const theCase = request.body as caseApiV1.CasePostRequest;
+      const rawCase = request.body as caseApiV1.CasePostRequest;
+      const theCase = emptyCaseAssigneesSanitizerV1(rawCase);
 
-      const res: caseDomainV1.Case = await casesClient.cases.create({ ...theCase });
+      const res: caseDomainV1.Case = await casesClient.cases.create(theCase);
```

Comment:
not sure why `theCase` was spread before but as theCase is now a new object anyways I though it would be ok to pass it through this way